src/Altinn.App.Core/Models/Components.cs

```diff
@@ -8,6 +8,6 @@ public class Components
         /// <summary>
         /// Exclude from pdf
         /// </summary>
-        public List<string> ExcludeFromPdf { get; set; }
+        public List<string>? ExcludeFromPdf { get; set; }
```

Comment:
I sometimes prefer this approach too, but this class is the target of Json Deserialization of a file that users will edit in VSCode. I think it is better to include the nullability in the classes, instead of pretending that we don't support json that we actually need to support for backwards compatibility.

Reply:
Ah, I missed that one! I will mark these as resolved.